docs/data-access/data-download.md

```diff
@@ -0,0 +1,89 @@
+- The score-client will automatically resume downloads if interrupted or paused briefly
+
+## Searching for Files
+
+Use the Platform [File Repository](https://platform.icgc-argo.org/repository) to search for a file set of interest. Query results can be narrowed down by selecting specific file values in the facet filters on the left side of the page. Once you have a file set identified, click `Download > File Manifest` on the top right of the table to download a TSV file manifest.
+
+The file manifest will contain a listing of the files that matched your query, along with some additional metatdata to assist in file identification. This file manifest will be used by the score-client as a list of files to download.
+
+> NOTE: Clinical data can be downloaded by any user. In order to download controlled molecular data, you **must have ICGC DACO approval**. Learn more about the [DACO application process here](data-access), or [apply for DACO approval here](https://icgc.org/daco).
+
+## Installing the Score-Client
+
+The score-client can be run in different ways depending on your operating system or setup:
+
+- If you are on Windows, use the Docker distribution
+- If you are on a Unix system (IOS/Linux) you can use the Docker distribution, or score-client directly.
+
+## Installing the Score-Client to Download Files
+
+### Docker Distribution
+
+Pull the latest version of the Score Docker distribution:
+
+```
+docker pull overture/score
+```
+
+Use the docker run command with the correct variables specified. You will need to define:
+
+- your personal [API Key](user-profile-and-token)
+- the file metadata Song server URL
+- the object storage Score server URL
+- the absoloute directory path where your file manifest is located
+- an output directory path
+
+```
+docker run --rm -it -e "METADATA_URL=https://song.argo.cancercollaboratory.org" -e "STORAGE_URL=https://score.argo.cancercollaboratory.org" -e "ACCESSTOKEN=92038829-338c-4aa2-92fc2-a3c241f63ff0" -v "C:\Users\username\Desktop\directory-path\" overture/score:latest score-client download --manifest /directory-path/score-manifest.20200520.tsv --output-dir C:\Users\rbajari\Desktop\download\
+```
+
+### Score-client with Configured Values
+
+Download the **[latest version of the Score-client](https://artifacts.oicr.on.ca/artifactory/dcc-release/bio/overture/score-client/[RELEASE]/score-client-[RELEASE]-dist.tar.gz)**.
+Once you have unzipped the tarball, update the `/conf/application.properties` configuration file with the correct user values, including:
+
+- your personal [API Key](user-profile-and-token)
+- the file metadata Song server URL
+- the object storage Score server URL
+
+This is an example of how your `application.properties` configuration file should look:
+
+```yaml
+# The access token for authorized access to data
+accessToken=92038829-338c-4aa2-92fc2-a3c241f63ff0
+
+# The location of the metadata service (SONG)
+metadata.url=https://song.argo.cancercollaboratory.org
```

Comment:
will they ever change these urls? If not, maybe just specify that they need to just put in their API Key

Reply:
- 1. well, the client does not come with these initialized so they HAVE to update them or it wont work
- 2. they might change them to point to different repositories

Reply:
Just not sure how they know these urls..but if you think it's clear, then leave it

Reply:
yeah we need to establish data locations - these urls will be the only ones for now, but in future there could be more! we will need extra documentatin next year after we establish mroe rdpcs

Reply:
if they are the only urls right now, then maybe don't say they have to customize them yet?

Reply:
but they do bc the default urls in the score client will not be correct